io.github.limo_app.limo.json

```diff
@@ -0,0 +1,345 @@
+            "tag": "0.26.0"
+          },
+          "dependencies/cbindgen-sources.json"
+      ],
+      "cleanup":
+      [
+        "*"
+      ]
+    },
+    {
+      "name": "yamlcpp",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DYAML_MSVC_SHARED_RT=ON",
+        "-DYAML_CPP_BUILD_CONTRIB=OFF",
+        "-DYAML_CPP_BUILD_TOOLS=OFF",
+        "-DYAML_BUILD_SHARED_LIBS=OFF",
+        "-DYAML_CPP_BUILD_TESTS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "archive",
+          "url": "https://github.com/loot/yaml-cpp/archive/0.8.0+merge-key-support.2.tar.gz",
+          "sha256": "e91779044cd4025ec10ed31986031031ef22ebce90a64cc8f85e7c7e95a1abc6"
+        }
+      ],
+      "cleanup":
+      [
+        "*"
+      ]
+    },
+    {
+      "name": "spdlog",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/gabime/spdlog.git",
+          "tag": "v1.14.1"
+        }
+      ],
+      "cleanup":
+      [
+        "*"
+      ]
+    },
+    {
+      "name": "libloot",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DLIBLOOT_BUILD_TESTS=OFF",
+        "-DLIBLOOT_INSTALL_DOCS=OFF",
+        "-DESPLUGIN_URL=/run/build/libloot/6.1.0.tar.gz",
+        "-DLIBLOADORDER_URL=/run/build/libloot/17.0.1.tar.gz",
+        "-DLOOT_CONDITION_INTERPRETER_URL=/run/build/libloot/4.0.1.tar.gz"
+      ],
+      "build-options":
+      {
+        "env":
+        {
+          "CARGO_HOME": "/run/build/libloot/cargo"
+        }
+      },
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/loot/libloot.git",
+          "tag": "0.23.0"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/Ortham/esplugin/archive/refs/tags/6.1.0.tar.gz",
+          "sha256": "52dec796d98426e1c75c5cf3c41351c0b3431a6301137e00fcf6023f4e47b502"
+        },
+        "dependencies/esplugin-sources.json",
+        {
+          "type": "file",
+          "url": "https://github.com/Ortham/libloadorder/archive/refs/tags/17.0.1.tar.gz",
+          "sha256": "36d90eb5cd53cbbf9d04de7ead561d69ea6aadcf3363d425a3c8370dc566c1b6"
+        },
+        "dependencies/libloadorder-sources.json",
+        {
+          "type": "file",
+          "url": "https://github.com/loot/loot-condition-interpreter/archive/refs/tags/4.0.1.tar.gz",
+          "sha256": "12bf0b108e4734baa3634e98cd7952a9411352539be3ec270aeee97c1b8db92f"
+        },
+        "dependencies/loot-condition-interpreter-sources.json"
+      ]
+    },
+    {
+      "name": "unrar",
+      "buildsystem": "simple",
+      "build-commands":
+      [
+        "make lib",
+        "make install-lib DESTDIR=/app",
```

Comment:
This should be buildsystem: make, you can use `make-install-args` and `make-args` to pass arguments.

The two commands below should be in `post-install`.

Reply:
Sorry for all the extra work i'm causing but i can't seem to get make as a buildsystem to work:
Error: module unrar: Invalid buildsystem: "make"
I've moved the 2 commands to post-install.

Reply:
Ah yea my bad. I was thinking about buildstream...

This should work:

```json
        {
            "name": "unrar",
            "no-autogen": true,
            "no-make-install": true,
            "make-args": [
                "lib"
            ],
            "post-install": [
                "install -D dll.hpp /app/include/unrar/dll.hpp",
                "install -D libunrar.a /app/lib/libunrar.a"
            ],
            "sources": [
                {
                    "type": "git",
                    "url": "https://github.com/aawc/unrar.git",
                    "tag": "v7.0.9"
                }
            ]
        }
```

Reply:
That works, thanks